app/forms/two_factor_authentication/phone_deletion_form.rb

```diff
@@ -0,0 +1,52 @@
+module TwoFactorAuthentication
+  class PhoneDeletionForm
+    include ActiveModel::Model
+
+    attr_reader :user, :configuration
+
+    def initialize(user, configuration)
+      @user = user
+      @configuration = configuration
+    end
+
+    def submit
+      success = configuration_absent? ||
```

Comment:
What do you think about making these validations for consistency with other forms? Also, should we localize the error messages?